Controllers/state_controller.py

```diff
@@ -48,6 +48,15 @@ def create_new_window(self, session_name, table_name="Default Title", video=None
 
         self.global_settings_manager.load_global_settings()
 
+        # After loading the values from QSettings, adjusts the table format.
+        self.window.table_panel.table.set_cell_size(
```

Comment:
These lines should cause exceptions since we're passing in strings, when the methods are expecting numerical data. I believe the comments in global_settings_manager should address these issues.